fancy-weather/src/index.js

```diff
@@ -0,0 +1,426 @@
+    localStorage.setItem('scale', 'celsium');
+  } else {
+    currentScale = localStorage.getItem('scale');
+  }
+  toggleTemperatureButton();
+}
+
+function getLocationByIP() {
+  return fetch('https://ipinfo.io/json?token=72f4ef7fa30822')
+    .then((response) => response.json())
+    .then((jsonResponse) => {
+      console.log(jsonResponse);
+      coordinatesIP = jsonResponse.loc;
+      return data;
+    });
+}
+
+function getCoordinates(query) {
+  return fetch(`https://api.opencagedata.com/geocode/v1/json?q=${query}&key=7dd8ba7a84bd4d0fa897bf84577d8693&language=${currentLanguage}&no_annotations=1&limit=1`)
+    .then((response) => response.json())
+    .then((jsonResponse) => {
+      console.log(jsonResponse);
+      data.geocoding.city = jsonResponse.results[0].components.city || jsonResponse.results[0].components.town || jsonResponse.results[0].components.village;
+      data.geocoding.country = jsonResponse.results[0].components.country;
+      data.geocoding.latitude = jsonResponse.results[0].geometry.lat;
+      data.geocoding.longitude = jsonResponse.results[0].geometry.lng;
+      return data;
+    });
+}
+
+function loadWeather(data) {
+  return fetch(`https://api.openweathermap.org/data/2.5/onecall?lat=${data.geocoding.latitude}&lon=${data.geocoding.longitude}&exclude=minutely,hourly,alerts&units=metric&lang=${currentLanguage}&appid=43aa3c58f07da360c90f88e6c11cdac3`)
+    .then((response) => response.json())
+    .then((jsonResponse) => {
+      console.log(jsonResponse);
+      data.time.offset = jsonResponse.timezone_offset / 3600;
+      data.weather.currentTemp = jsonResponse.current.temp;
+      data.weather.scale = 'celsium';
+      data.weather.currentDescription = jsonResponse.current.weather[0].description;
+      data.weather.currentFeelsLike = jsonResponse.current.feels_like;
+      data.weather.currentWindSpeed = jsonResponse.current.wind_speed;
+      data.weather.currentHumidity = jsonResponse.current.humidity;
+      data.weather.currentIcon = jsonResponse.current.weather[0].icon;
+      data.weather.nextFirstDayTemp = jsonResponse.daily[1].temp.day;
+      data.weather.nextFirstDayIcon = jsonResponse.daily[1].weather[0].icon;
+      data.weather.nextSecondDayTemp = jsonResponse.daily[2].temp.day;
+      data.weather.nextSecondDayIcon = jsonResponse.daily[2].weather[0].icon;
+      data.weather.nextThirdDayTemp = jsonResponse.daily[3].temp.day;
+      data.weather.nextThirdDayIcon = jsonResponse.daily[3].weather[0].icon;
+      return data;
+    });
+}
+
+function determineTimeOfDay() {
+  switch (true) {
+    case data.time.hour >= 5 && data.time.hour <= 11:
+      timeOfDay = 'morning';
+      break;
+    case data.time.hour >= 18 && data.time.hour <= 21:
+      timeOfDay = 'evening';
+      break;
+    case (data.time.hour > 21 && data.time.hour <= 24) || (data.time.hour >= 0 && data.time.hour < 5):
+      timeOfDay = 'night';
+      break;
+    case data.time.hour > 11 && data.time.hour < 18:
```

Comment:
@andreiha 
Try to store such number (18, 21, 24, 5, 11) to const/config with association name
So, now it's look like as s magic number